exporters/otlp-http/trace/src/main/java/io/opentelemetry/exporter/otlp/http/trace/OtlpHttpSpanExporter.java

```diff
@@ -142,6 +167,28 @@ public void onResponse(Call call, Response response) {
     return result;
   }
 
+  private boolean isSuccessful(Response response) {
+    if (!response.isSuccessful()) {
+      return false;
+    }
+
+    if (!useGrpc) {
+      return true;
+    }
+
+    // Status can either be in the headers or trailers depending on error
+    String grpcStatus = response.header(GRPC_STATUS);
+    if (grpcStatus == null) {
+      try {
+        grpcStatus = response.trailers().get(GRPC_STATUS);
```

Comment:
By the way, I had an idea of trying out a completely no-deps artifact for Java 11. But then I found this 😂 

https://github.com/openjdk/jdk/blob/master/src/java.net.http/share/classes/jdk/internal/net/http/HttpResponseImpl.java#L65

Reply:
Well that's unfortunate. 😬